### PR TITLE
Feat: change the pagination to cursor and add sorter --- Bridge Token and Layer 2 Native Token

### PR DESCRIPTION
**Pull request by JeffreyMa597**
What problem does this PR solve?
------
This PR is for the page of `Bridge Token` and `Layer 2 Native Token`.

Ref: https://github.com/Magickbase/godwoken_explorer/issues/556

Check List
------
1、sorted by total supply/circulating supply
2、sorted by holders
3、sorted by name
4、filtered by name (fuzzy)

### Test
e2e Test
### Task
none

### CODE REVIEW

**Comment by FrederLu** [comment]
Click the sorted by total supply/circulating supply/name page data to remain unchanged.
image
image

File: pages/tokens/styles.module.scss [hunk 1]
```diff
@@ -76,3 +76,11 @@
 .noRecords {
   @include empty-list;
 }
+
+.pr-4 {
```
Comment (Keith-CY) [inline pages/tokens/styles.module.scss:80]: Use a semantic tab instead of an atomic tag

**Comment by Keith-CY** [comment]
@JeffreyMa597 any update on this feature

**Comment by JeffreyMa597** [comment]
> @JeffreyMa597 any update on this feature

I've changed this tag, but I can't upload my code. I'm still focusing on my network problem.

**Review by Keith-CY — APPROVED** [review]

**Pull request merged**